pyqtgraph/graphicsItems/AxisItem.py

```diff
@@ -903,7 +919,10 @@ def generateDrawSpecs(self, p):
         be drawn, then generates from this a set of drawing commands to be
         interpreted by drawPicture().
         """
-        profiler = debug.Profiler()
+        profiler = debug.Profiler(
```

Comment:
we're not going to want to leave these settings here in the library or else it would print to everyone in the console; a safer alternative for testing might be to try and setup `.env` file with `PYQTGRAPHPROFILE=pyqtgraph.graphicsItems.AxisItem.setStyle` (I think that should work, haven't tested, the docstring for the Profiler has the blurb about setting environment variables).

Reply:
Yah of course. This patch is **no where near meant for landing**.

This is just to get some peeps testing and see if they figure if the cache usage also speeds up their systems.